documents/EmotionConfig Documentation.tex

```diff
@@ -0,0 +1,76 @@
+\usepackage{color}
+%footer
+\usepackage{fancyhdr}
+\pagestyle{fancy}
+\fancyhead{}
+\fancyfoot[R]{Context Project: Gamygdala-Integration}
+\renewcommand{\headrulewidth}{0pt}
+\renewcommand{\footrulewidth}{0.4pt}
+
+\begin{document}
+
+\title{Documentation of the EmotionConfig}
+\subtitle{Group: Gamygdala-Integration}
+\date{\today{}}
+
+\author{
+    \begin{tabular}{l r}
+      B.L.L. Kreynen\\
+      M. Spanoghe\\
+      R.A.N. Starre\\
+      Yannick Verhoog\\
+      Joost Wooning\\
+    \end{tabular}
+}
+
+\maketitle \thispagestyle{empty} \pagebreak
+
+\section*{Commands to set specific values}
+Here you can find all the commands that can be used to set certain default values to your own needs. Keep in minds that the strings (the first part of each command) can also be written in small letters since they will be transformed to uppercases automatically.
+
+\subsection*{"DEFAULT GOAL UTILITY": String, value: double}
+This sets the default goal utility to a specific value. The value is a double.\\
+Example line in text file:\\
+DEFAULT GOAL UTILITY,0.5\\
+
+\subsection*{"DEFAULT ACHIEVED CONGRUENCE": String, value: double}
+This sets the default congruence for goals that are achieved. It specifies how "good" your default feedback is when you achieve a goal.
+Example line in text file:\\
+DEFAULT ACHIEVED CONGRUENCE,1\\
+
+\subsection*{"DEFAULT DROPPED CONGRUENCE": String, value: double}
+This sets the default congruence for goals that are NOT achieved. It specifies how "good" your default feedback is when you can't achieve a goal.The default value is still expressed as how "good" it is to drop goals. Therefore keep in mind that when you want to have a negative feedback that you have to put in a negative number.\\
+Example line in text file:\\
+DEFAULT DROPPED CONGRUENCE,-0.2\\
+
+\subsection*{"DEFAULT BELIEF LIKELIHOOD": String, value: double}
+This rule specifies the default setting for the likelihood of beliefs. This value tells thus how "likely" belief are to be true when using this default setting.\\
+Example line in text file:\\
+DEFAULT BELIEF LIKELIHOOD,0.9\\
+
+\subsection*{"DEFAULT ISINCREMENTAL": String, value: Boolean}
+This specifies if the beliefs of goals are incremented or not. See documentation of Gamygdala for more information on what this feature does.
+You can write any upper/lower case variant of true/false. (True,False,true,false,fAlse...). (The value is being transformed to uppercases)
+Writing anything wrong in the syntax will throw an error instead of setting it on false by default.\\
+Example line in text file:\\
+DEFAULT ISINCREMENTAL, true\\
+
+\pagebreak
+\section*{default values}
+When you don't specify these commands in the EmotionConfig default values will be used. Here are the settings that will be present if you don't specify anything.\\
+DefaultUtility = 1;\\
+DefaultNegativeCongruence = -0.1;\\
+DefaultPositiveCongruence = 0.5;\\
+DefaultBelLikelihood = 1;\\
+DefaultIsIncremental = false;\\
+
```

Comment:
Verandert naar files changed: zet hier even de namen die gebruikt worden om ze te definiëren in de emotionconfig ipv de namen van de variablen in de source code.